src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/ocrvalidation/OcrValidationClientTest.java

```diff
@@ -0,0 +1,104 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation;
+
+import com.github.tomakehurst.wiremock.core.Options;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.client.HttpServerErrorException.InternalServerError;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationContextInitializer;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.IntegrationTest;
+import uk.gov.hmcts.reform.bulkscanprocessor.config.Profiles;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.OcrValidationClient;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.FormData;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.req.OcrDataField;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.res.Status;
+import uk.gov.hmcts.reform.bulkscanprocessor.ocrvalidation.client.model.res.ValidationResponse;
+
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.okJson;
+import static com.github.tomakehurst.wiremock.client.WireMock.post;
+import static com.github.tomakehurst.wiremock.client.WireMock.serverError;
+import static com.github.tomakehurst.wiremock.client.WireMock.stubFor;
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+@ActiveProfiles( {
+    IntegrationContextInitializer.PROFILE_WIREMOCK,
+    Profiles.SERVICE_BUS_STUB,
+    Profiles.STORAGE_STUB
+})
+@AutoConfigureWireMock
+@IntegrationTest
+@RunWith(SpringRunner.class)
+public class OcrValidationClientTest {
```

Comment:
Could we please add tests when there is success and when there are only warnings ? and also for invalid s2s token and 403 scenarios ?

Reply:
done